src/operator/csv.rs

```diff
@@ -0,0 +1,110 @@
+//! Source operator that reads data from a CSV file.
+#![cfg(feature = "with-serde")]
```

Comment:
This should be `with-csv`, otherwise it's broken if only `with-serde` is enabled